style_guides/scss_style_guide.md

```diff
@@ -15,6 +15,10 @@ Our style guide is an extension of [Sass Guidelines by Hugo Giraudel](https://sa
 The Sass Guidelines site recommends using RBG and HSL values to format colors, but we're using
 hex values.
 
+### Bemify for namespacing
+
+We use the [bemify](https://github.com/franzheidl/bemify) for namespacing the sass into a BEM format. We use `component`, `child`, `modifier` and `state` as our mixin names. We've adjusted the plugin's state mixin so that you need to write out the full selector (`@include state('is-happening')`).
```

Comment:
Can we add an example to demonstrate how each mixin should be used? This should also help address some of @kjbekkelund's questions about why/what, if they occur to anyone else.